h2o-docs/src/product/data-science/generic.rst

```diff
@@ -0,0 +1,98 @@
+Generic Models
+--------------
+
+Introduction
+~~~~~~~~~~~~
+
+A Generic model provides a means to use external, pre-trained models in H2O - mainly for the purpose of scoring. Depending on each external model, metrics and other model information might be obtained as well. Currently, only selected H2O MOJOs are supported. (See the :ref:`mojo_quickstart` section for information about creating MOJOs.)
+
+Supported MOJOs
+~~~~~~~~~~~~~~~
+
+Only a subset of H2O MOJO models is supported in this version. 
+
+-  GBM (Gradient Boosting Machines)
+-  DRF (Distributd Random Forest)
+-  IRF (Isolation Random Forest)
+-  GLM (Generalized Linear Model)
+
+Importing a Generic Model
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Importing a Generic model is available from Python, R, and Flow. As the model is generic, there are no hyperparameters and there is no real training phase for Generic models. H2O imports the model and embraces it for the purpose of scoring. Information output about the model may be limited.
+
+Importing in R or Python
+''''''''''''''''''''''''
+
+.. example-code::
+   .. code-block:: r
+
+    data <- h2o.importFile(path = 'training_dataset.csv')
+    cols <- c("Some column", "Another column")
+    model.original <- h2o.glm(x=cols, y = "response", training_frame = data)    
+
+    path <- "/path/to/model/directory"
+    mojo.destination <- h2o.download_mojo(model = model.original, path = path)
+    model.generic <- h2o.genericModel(mojo.destination)
```

Comment:
My only comment would be to not use dots in object names (this goes against the [tidyverse style guide](https://style.tidyverse.org/), so I'd change this to `model_original`, `mojo_destination` and `model_generic`.

Reply:
Also ideally we would use the same object names in the R and Python examples (e.g. `glm` and `model` in Python vs `model_original` and `model_generic` in R) when we do side-by-side examples in the docs.

Reply:
Resolved. Dots removed, variable names aligned.